docs/source/ledger-nano.rst

```diff
@@ -0,0 +1,261 @@
+
+Prerequisites
+-------------
+
+To use the Insolar application on your Ledger Nano S or X hardware wallet, make sure to install and set up the following:
+
+#. `Install Google Chrome <https://www.google.com/chrome/>`_. It's the only supported browser for now.
+#. `Install the Ledger Live application <https://support.ledger.com/hc/en-us/articles/360006395553/>`_. It allows you to manage applications on your device.
+#. If you don't yet have an Insolar Wallet, set up your `Nano S <https://support.ledger.com/hc/en-us/articles/360000613793>`_ or `Nano X <https://support.ledger.com/hc/en-us/articles/360018784134>`_.
+
+   .. important::
+
+     During the setup, you are required to choose a PIN and write down the recovery phrase. Without the PIN you will not be able to unlock the device and without the recovery phrase—restore access to your wallet.
+
+   Otherwise, if you do have an Insolar Wallet, initialize your device with the existing backup phrase:
+
+   #. Follow the official instructions for `Nano S <https://support.ledger.com/hc/en-us/articles/360005434914>`_ or `Nano X <https://support.ledger.com/hc/en-us/articles/360015132494>`_ to initialize the wallet with the phrase.
+   #. At **step 3**, choose the 18-word length and enter your Insolar Wallet's backup phrase.
+   
+#. (Ledger Nano S only) `Install the latest firmware on your device <https://support.ledger.com/hc/en-us/articles/360002731113-Update-Ledger-Nano-S-firmware>`_.
+
+Next, install the Insolar application on your device as described below.
+
+.. _install-ins-app:
+
+Install the Insolar application on Ledger Nano
+----------------------------------------------
+
+To install the Insolar application on Ledger Nano:
+
+#. Open Ledger Live, go to :guilabel:`Settings` > :guilabel:`Experimental features`, and enable :guilabel:`Developer mode`.
+
+   .. image:: imgs/mig-test/ledger-dev-mode.png
+      :width: 700px
+
+#. Open the :guilabel:`Manager` tab and connect and unlock your Ledger Nano.
+
+   .. image:: imgs/mig-test/ledger-live-connect.png
+      :width: 600px
+
+#. If prompted, press both :guilabel:`left` and :guilabel:`right` buttons simultaneously on the device to allow the manager connection.
+
+   .. image:: imgs/mig-test/allow-ledger-live.png
+      :width: 300px
+
+#. Find :guilabel:`Insolar` in the application catalog and click :guilabel:`Install` next to it.
+
+   This displays the installation window with a progress bar. Wait for the installation to complete.
+
+#. In the dashboard of the Ledger Nano device, press :guilabel:`left` or :guilabel:`right` buttons to find the Insolar application.
+
+#. Once found, press both :guilabel:`left` and :guilabel:`right` buttons simultaneously to launch the application.
+
+Once the Insolar application is launched, proceed to creating an Insolar Wallet if you don't have one. Otherwise, :ref:`log in your wallet <log-in-nano>`.
```

Comment:
Once the Insolar application is launched, proceed to creating an Insolar Wallet if you don't have one. Otherwise, :ref:`log in **to** your wallet <log-in-nano>`.